test/ring/middleware/authorization_test.clj

```diff
@@ -0,0 +1,71 @@
+(ns ring.middleware.authorization-test
+  (:require [clojure.test :refer :all]
+            [ring.middleware.authorization :as mw]))
+
+(deftest test-parse-auth-params
+  (is (= (#'mw/parse-auth-params "A=B, c=\"d\", eeee=\"dGVzdA==\", fparam=dGVzdA==, g, \"h\"=i, =j, = ,, , k=1")
```

Comment:
Unfortunately we can't test private functions directly, as the tests should still work even if we refactor the internals. We should test these via `authorization-request`.